concepticondata/conceptlists/Bowern-2021-207a.tsv

```diff
@@ -0,0 +1,208 @@
+Bowern-2021-207a-58	58	dust	401	environment	Basic Vocabulary	2	DUST
+Bowern-2021-207a-59	59	ear	402	body	Basic Vocabulary	1247	EAR
+Bowern-2021-207a-60	60	earth/soil	403	environment	Basic Vocabulary	1228	EARTH (SOIL)
+Bowern-2021-207a-61	61	eat	404	body	Basic Vocabulary	1336	EAT
+Bowern-2021-207a-62	62	egg	405	fauna	Basic Vocabulary	744	EGG
+Bowern-2021-207a-63	63	eye	406	body	Basic Vocabulary	1248	EYE
+Bowern-2021-207a-64	64	faeces	407	body	Basic Vocabulary	675	FAECES (EXCREMENT)
+Bowern-2021-207a-65	65	fall	408	motion	Basic Vocabulary	1280	FALL
+Bowern-2021-207a-66	66	far	409	quality	Basic Vocabulary	1406	FAR
+Bowern-2021-207a-67	67	fat/grease	410	body	Basic Vocabulary	323	FAT (ORGANIC SUBSTANCE)
+Bowern-2021-207a-68	68	father	411	kinship	Basic Vocabulary	1217	FATHER
+Bowern-2021-207a-69	69	fear	412	mental	Basic Vocabulary	781	FEAR (FRIGHT)
+Bowern-2021-207a-70	70	feather	413	fauna	Basic Vocabulary	1201	FEATHER
+Bowern-2021-207a-71	71	fingernail	821	body	Basic Vocabulary	1258	FINGERNAIL
+Bowern-2021-207a-72	72	fire	414	environment	Basic Vocabulary	221	FIRE
+Bowern-2021-207a-73	73	fish	415	fauna	Basic Vocabulary	227	FISH
+Bowern-2021-207a-74	74	flow	416	motion	Basic Vocabulary	2003	FLOW
+Bowern-2021-207a-75	75	flower	278	flora-fauna	Basic Vocabulary	239	FLOWER
+Bowern-2021-207a-76	76	flower	417	environment	Basic Vocabulary	239	FLOWER
+Bowern-2021-207a-77	77	fly	418	motion	Basic Vocabulary	1441	FLY (MOVE THROUGH AIR)
+Bowern-2021-207a-78	78	fog	419	environment	Basic Vocabulary	249	FOG
+Bowern-2021-207a-79	79	foot	420	body	Basic Vocabulary	1301	FOOT
+Bowern-2021-207a-80	80	fruit	421	flora	Basic Vocabulary	1507	FRUIT
+Bowern-2021-207a-81	81	good	422	quality	Basic Vocabulary	1035	GOOD
+Bowern-2021-207a-82	82	green	825	quality	Basic Vocabulary	1425	GREEN
+Bowern-2021-207a-83	83	grow	424	state	Basic Vocabulary	479	GROW
+Bowern-2021-207a-84	84	hair (of head)	425	body	Basic Vocabulary	2648	HAIR (HEAD)
+Bowern-2021-207a-85	85	hand	426	body	Basic Vocabulary	1277	HAND
+Bowern-2021-207a-86	86	head	428	body	Basic Vocabulary	1256	HEAD
+Bowern-2021-207a-87	87	hear	429	mental	Basic Vocabulary	1408	HEAR
+Bowern-2021-207a-88	88	heart	822	body	Basic Vocabulary	1223	HEART
+Bowern-2021-207a-89	89	heavy	430	quality	Basic Vocabulary	1210	HEAVY
+Bowern-2021-207a-90	90	hide	431	motion	Basic Vocabulary	602	HIDE (CONCEAL)
+Bowern-2021-207a-91	91	hit	432	impact	Basic Vocabulary	1433	HIT
+Bowern-2021-207a-92	92	hit (with hand)	576	impact	Basic Vocabulary		
+Bowern-2021-207a-93	93	hold	433	other	Basic Vocabulary	1448	HOLD
+Bowern-2021-207a-94	94	hot	826	quality	Basic Vocabulary	1286	HOT
+Bowern-2021-207a-95	95	how?	435	grammar	Basic Vocabulary	1239	HOW
+Bowern-2021-207a-96	96	if	437	grammar	Basic Vocabulary	1459	IF
+Bowern-2021-207a-97	97	in/inside	438	location	Basic Vocabulary	1606	INSIDE
+Bowern-2021-207a-98	98	intestines	439	body	Basic Vocabulary	1991	INTESTINES
+Bowern-2021-207a-99	99	itch	440	body	Basic Vocabulary	3855	ITCH (CAUSE ITCHING OR FEEL ITCHY)
```

Comment:
148	ITCH ?

Reply:
The Spanish gloss indicates the verbal meaning, if I'm not mistaken.